packages/opentelemetry-plugin-winston/src/utils.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TraceInformation, TRACE_PARAM_NAME, WinstonChunk } from './types';
+import * as types from '@opentelemetry/types';
+
+/**
+ * Get trace information from span
+ * @param span
+ */
+function getTraceInformation(span: types.Span): TraceInformation {
+  const traceInformation: TraceInformation = {
+    trace_id: span.context().traceId,
+    span_id: span.context().spanId,
+  };
+  return traceInformation;
+}
+
+/**
+ * Adds information to chunk about trace
+ * @param chunk
+ * @param span
+ */
+export function addTraceToChunk(
+  chunk: WinstonChunk,
+  span: types.Span
+): WinstonChunk {
+  const newChunk: WinstonChunk = Object.assign({}, chunk);
+  newChunk[TRACE_PARAM_NAME] = getTraceInformation(span);
+  return newChunk;
+}
+
+/**
+ * Process arguments and adds additional information about trace
+ * @param args
+ * @param span
+ */
+export function processArgs(args: any[], span: types.Span): any[] {
```

Comment:
Can you make this name something more specific than "process"?

Reply:
pls make suggestion for name (for me the `processArgs` is fine)

Reply:
The function name, type, and comment gives me little to no idea what this function does, what arguments should be passed to it, or what is returned. Reading the code it looks like `args` is actually `chunks`? It still doesn't really tell me what a chunk is. The argument `args: any[]` seems to imply I can sort of just pass whatever I want into here?